contrib/README.md

```diff
@@ -0,0 +1,24 @@
+# `contrib`
+
+Contributed examples of how users may use the `easage` library and tools.
+
+---
+
+## `easage-pack.cmd` and `easage-unpack.cmd`
```

Comment:
I like `unpack` over `extract`. I'll change other bits of the project to use that wording after this is merged.

Reply:
I was thinking of pitching that in because it makes sense to have `unpack` to be reverse of `pack`.